<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AthScan</class>
 <widget class="QMainWindow" name="AthScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AthScan</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QwtTextLabel" name="minXLabel">
        <property name="plainText">
         <string>Min Freq [MHz]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minFreqSpinBox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>2400</number>
        </property>
        <property name="maximum">
         <number>6000</number>
        </property>
        <property name="singleStep">
         <number>20</number>
        </property>
        <property name="value">
         <number>2400</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtTextLabel" name="maxXLabel">
        <property name="plainText">
         <string>Max Freq [MHz]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxFreqSpinBox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>2400</number>
        </property>
        <property name="maximum">
         <number>6000</number>
        </property>
        <property name="singleStep">
         <number>20</number>
        </property>
        <property name="value">
         <number>6000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtTextLabel" name="minYLabel">
        <property name="plainText">
         <string>Min Pwr [db]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minPwrSpinBox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>-96</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>-96</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtTextLabel" name="maxYLabel">
        <property name="plainText">
         <string>Max Pwr [db]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxPwrSpinBox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>-96</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QwtPlot" name="fftPlot">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
